pkgs/top-level/all-packages.nix

```diff
@@ -226,6 +226,8 @@ with pkgs;
       }
     '');
 
+  checkMeta = callPackage ../stdenv/generic/check-meta.nix { };
```

Comment:
Why this does not trigger by-name checks?

Reply:
No derivations added I guess?